scss/civicrm/administer/events/_badgelayout.scss

```diff
@@ -0,0 +1,23 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+&#{civi-page('admin-badgelayout')} {
+  .crm-container {
+    h3 {
+      @include box-shadow($box-shadow);
+      @include panel-header;
+      margin-bottom: 0;
+    }
+
+    .crm-badge-layout-form-block {
+      @include box-shadow(0 8px 18px 0 rgba(48,40,40,0.25));
```

Comment:
shouldn't we have one value for box shadow across the whole theme?

Reply:
This is slightly different version than standard box shadow, we use it on few pages. it's used when we want to combine detached section, to look like one.
I will create a variable for that as now, it's being use on more than one place